src/m365/spo/commands/listitem/listitem-get.ts

```diff
@@ -89,7 +89,7 @@ class SpoListItemGetCommand extends SpoCommand {
         option: '-t, --listTitle [listTitle]'
       },
       {
-        option: '-f, --fields [fields]'
+        option: '-f, --properties [properties]'
```

Comment:
We should use `-p` instead of `-f` to make a more intuitive short for `properties`